# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoader.cpp
# Commit: a50f60a612d3
# Full Hash: a50f60a612d31de79369e59933242ee10a5d810f
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move mElement from ScriptFetchOptions to DOMScriptLoadContext; r=jonco,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D136742
# ==============================================================================

diff -r 0ac6d9d26e2b -r a50f60a612d3 dom/script/ModuleLoader.cpp
--- a/dom/script/ModuleLoader.cpp	Thu Feb 17 15:57:48 2022 +0000
+++ b/dom/script/ModuleLoader.cpp	Thu Feb 17 15:57:48 2022 +0000
@@ -267,7 +267,8 @@
     }
 
     RefPtr<ModuleScript> moduleScript =
-        new ModuleScript(aRequest->mFetchOptions, aRequest->mBaseURL);
+        new ModuleScript(aRequest->mFetchOptions, aRequest->mBaseURL,
+                         aRequest->mLoadContext->mElement);
     aRequest->mModuleScript = moduleScript;
 
     if (!module) {